<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBWindowGlazingSystemEditWidget</class>
 <widget class="QWidget" name="SVDBWindowGlazingSystemEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window Glazing System Database</string>
  </property>
  <layout class="QGridLayout" name="masterLayout">
   <item row="4" column="2" colspan="2">
    <widget class="QToolButton" name="toolButtonCreateSpline">
     <property name="text">
      <string>Create Standard SHGC Spline</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelFrameUValue_2">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SHGC (angle dependent)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="labelSHGC">
     <property name="text">
      <string>SHGC (normal Incidence):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelDisplayName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditUValue"/>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="labelSHGCUnit">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelFrameUValue">
     <property name="text">
      <string>UValue:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditSHGC"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="comboBoxType"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelFrameInputUnit">
     <property name="text">
      <string>W/m2K</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QtExt::ColorButton" name="pushButtonWindowColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidgetSHGC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="7" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QwtPlot" name="shgcPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header location="global">qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
